recipes/magic_enum/build.sh

```diff
@@ -0,0 +1,11 @@
+mkdir -p build
+cd build
+cmake -DCMAKE_INSTALL_PREFIX=${PREFIX} \
```

Comment:
Please pass `CMAKE_ARGS` variable: `cmake ${CMAKE_ARGS}`. It adds some variables defined by conda-forge, including `CMAKE_INSTALL_PREFIX` (which you don't have to pass then).

Reply:
I added `${CMAKE_ARGS}`, should I remove `CMAKE_INSTALL_PREFIX` and `CMAKE_PREFIX_PATH`?

Reply:
Merging as is

Reply:
Thanks.